src/chunk-stream.iced

```diff
@@ -1,17 +1,25 @@
 stream = require('stream')
+{make_esc} = require('iced-error')
 
 exports.ChunkStream = class ChunkStream extends stream.Transform
 
-  constructor : ({@transform_func, @block_size, @exact_chunking, @writableObjectMode, @readableObjectMode}) ->
+  constructor : ({@transform_func, @block_size, @readableObjectMode}) ->
     @extra = null
-    super({@writableObjectMode, @readableObjectMode})
+    super({@readableObjectMode})
+
+  # chunk must be guaranteed to be a multiple of the block size
+  _transform_chunk : (chunk, cb) ->
+    blocks = []
+    count = 0
+    for i in [0...chunk.length] by @block_size
+      block = chunk[i...i+@block_size]
+      await @transform_func(block, defer(err, blocks[count]))
```

Comment:
This `err` is dropped on the ground. But maybe we should get rid of this entire function anyway?